src/components/CustomFileInput/index.js

```diff
@@ -36,10 +36,39 @@ export default class CustomFileInput extends Component<Props> {
     event.currentTarget.value = '' // Allow re-uplading the same file
   }
 
+  handleDragOverEvent = (ev: DragEvent) => {
+    // Prevent default behavior (Prevent file from being opened)
+    ev.preventDefault();
+  }
```

Comment:
Is this still needed with the `handleDropEvent` also catching the event to prevent default?

Reply:
yes its needed otherwise file opens in browser

Reply:
Fair enough. Wasn't so sure about this one.